runtime/src/commitment.rs

```diff
@@ -41,7 +40,7 @@ pub struct BlockCommitmentCache {
     block_commitment: HashMap<Slot, BlockCommitment>,
     highest_confirmed_root: Slot,
     total_stake: u64,
-    bank: Arc<Bank>,
+    recent_slot: Slot,
```

Comment:
@CriesofCarrots, can you recommend a name here? `bank` didn't say anything about which bank. Same for the `slot()` method.  I guessed "recent", but maybe "highest" or "heaviest".  Not obvious to me what this data is used for exactly.

Reply:
It's the slot of the bank from which the data in this BlockCommitmentCache was aggregated.
"highest" and "heaviest" both imply things about how/when the aggregation is triggered (right now in ReplayStage::handle_votable_bank) that might not be true forever.

Part of me thinks it should just be `slot`, because it is "the" slot for this BlockCommitmentCache data. But if we want to be more descriptive, `aggregation_data_slot`?

Reply:
got it, with that context, `slot` makes sense to me, thank you